src/main/java/org/protege/editor/owl/client/LocalHttpClient.java

```diff
@@ -665,7 +665,7 @@ public void putEVSHistory(String code, String name, String operation, String ref
 		try {
 			History evsHistory = new History(userId.get(), code, name, operation, reference);
 			ByteArrayOutputStream b = writeRequestArgumentsIntoByteStream(evsHistory);
-			post(EVS_REC,
+			post(EVS_REC + "?projectid=" + projectId.get(),
```

Comment:
Note that adding query parameters is best done with `HttpUrl.Builder`. Although it might be difficult with the current API.  Perhaps we need an overload that accepts a `HttpUrl` directly?